src/app/onboarding/legal-experience/page.tsx

```diff
@@ -1,24 +1,97 @@
 'use client';
 
-import Button from '@/components/Button';
-import COLORS from '@/styles/colors';
+import { useContext, useState, useEffect, useCallback } from 'react';
 import { H1 } from '@/styles/text';
 import { OnboardingContext } from '@/utils/OnboardingProvider';
-import { useContext } from 'react';
+import TextInput from '@/components/TextInput';
+import InputDropdown from '@/components/InputDropdown';
+import RadioGroup from '@/components/RadioGroup';
+import { ImmigrationLawExperienceEnum } from '@/types/schema';
+
+const legalExperienceOptions = new Map<ImmigrationLawExperienceEnum, string>([
+  ['HIGH', 'Multiple cases of immigration law experience'],
+  ['MEDIUM', 'Few cases of immigration law experience'],
+  ['LOW', 'One or no case of immigration law experience'],
+]);
 
 export default function Page() {
   const onboarding = useContext(OnboardingContext);
+  const [barNum, setBarNum] = useState('');
+  const [registered, setRegistered] = useState('');
+  const [barNumError, setBarNumError] = useState('');
+
+  const isValidBarNumber = useCallback(
+    (barNumber: string) =>
+      !Number.isNaN(parseInt(barNumber, 10)) && /^\d{6}$/.test(barNumber),
```

Comment:
Here is what I can see from the California state bar website: https://apps.calbar.ca.gov/attorney/LicenseeSearch/QuickSearch?FreeText=Smith

It looks like it can go as low as 3 digits to 6 digits

Reply:
Definitely can be improved, but we need to check with the NPO on that. I think for California (our scope) though, it shouldn't be 6 or more.